src/shared/analytics/shared/mixpanel-data-helpers.ts

```diff
@@ -2,8 +2,8 @@ import type { Account } from 'src/background/account/Account';
 import { getAddressActivity } from 'src/ui/shared/requests/useAddressActivity';
 import { INTERNAL_SYMBOL_CONTEXT } from 'src/background/Wallet/Wallet';
 import { isReadonlyContainer } from 'src/shared/types/validators';
-import { backgroundQueryClient } from 'src/modules/query-client/query-client.background';
 import { ZerionAPI } from 'src/modules/zerion-api/zerion-api.background';
+import { backgroundQueryClient } from 'src/modules/query-client/query-client.background';
```

Comment:
weird diff

Reply:
yeah, merge side effects, looks like I picked the wrong line, but they are the same so...